public/data/projects.json

```diff
@@ -5,7 +5,7 @@
       "name": "CO2-Labels for University Canteen",
       "image": "/images/canteen.jpg",
       "organisation_name": "sneep Erlangen e.V.",
-      "organisation_image": "/images/sneep.png",
+      "organisation_image": "images/sneep.png",
```

Comment:
Should this be reverted?

Reply:
Oh got I keep screwing this up for some reason. Reverted it.